zebra/rt_netlink.c

```diff
@@ -755,9 +755,36 @@ static int netlink_route_change_read_unicast(struct nlmsghdr *h, ns_id_t ns_id,
 				nh.bh_type = bh_type;
 			}
 			nh.ifindex = index;
+
+			if (index) {
+				/*
+				 * Yes we are looking this up
+				 * for every nexthop and just
+				 * using the last one looked
+				 * up right now
+				 */
+				ifp = if_lookup_by_index_per_ns(
+								zebra_ns_lookup(ns_id),
```

Comment:
That indent looks weird to me: shouldn't `zebra_ns_lookup` be indented one position to the right of `if_lookup_xxx` instead of hanging where it is? the hanging effect just looks odd - it doesn't shorten the length of the line, for example?

Reply:
I used clang-format and just accepted what it spit out.

Reply:
well, I don't like to be a nit-picker, but ... does your editor accept this formatting and leave it as-is? I don't really understand some of the things that the auto-suggester-robot offers, and this is one of those patterns. so far, neither Quentin nor I have been able to figure out how to tell clang-format not to do this. introducing some pattern that is guaranteed to result in some other contributor's producing a white-space change if anyone else ever visits this code just seems ... not good. even if a robot offered the suggestion?